analyzer/consensus/consensus.go

```diff
@@ -621,7 +621,7 @@ func (m *Main) queueBurns(batch *storage.QueryBatch, data *storage.StakingData)
 }
 
 func (m *Main) queueEscrows(batch *storage.QueryBatch, data *storage.StakingData) error {
-	addGeneralBalanceUpdateQuery := m.qf.ConsensusAddGeneralBalanceUpdateQuery()
+	stakeGeneralBalanceUpdateQuery := m.qf.ConsensusDecreaseGeneralBalanceUpdateQuery()
```

Comment:
> `stakeGeneralBalanceUpdateQuery`

(nit) I dunno if we want to say it is sending or adding to escrow or something.

Reply:
sigh ... I was thinking the same. Updated to a long name.

On a related note, I dislike that we have `ConsensusBurnUpdateQuery` and `ConsensusDecreaseGeneralBalanceForEscrowUpdateQuery` that return _identical_ sql.

Either
1. The context from which they are called doesn't matter, and they should be a single method (`DecreaseGeneralBalance`) that happens to be called in multiple places (here, from escrow and from burn).
2. The context (here, putting funds into escrow) is important, and it doesn't make sense to have `ConsensusDecreaseGeneralBalanceForEscrowUpdateQuery` as a standalone function; instead, one function should generate all three SQL statements that need to happen to keep track of moving funds into escrow.

I like 2 because 1 results in these fragmented sql statements that are harder to make sense of (and write without bugs) without reading the callsites. What we have is mostly (there aren't many duplicated queries), but not entirely, 1.

Reply:
Agree that we mostly tried to follow 1 but not all the way. I'm also okay with 2.